tests/test_libplots.py

```diff
@@ -262,3 +287,67 @@ def test_offline_js_manager():
     plotly_cdn_full_url = plotly_cdn_url()
     offline_plotly_js = offline_js_manager(tmpdir, offline=False)
     assert plotly_cdn_full_url in offline_plotly_js
+
+
+def test_box_plot_handler(example_capri_ss, cluster_ranking):
+    """Test box plot generation without format definition."""
+    initdir = os.getcwd()
+    with tempfile.TemporaryDirectory() as tmpdir:
+        os.chdir(tmpdir)
```

Comment:
Not sure how these functions work, but can you avoid this (and the others) `chdir` or do they expect the files to be relative?

Reply:
- relative files
- easier to delete everything (will generate bunch of `.html` and `.png` files...)